.goreleaser.yml

```diff
@@ -33,5 +38,5 @@ changelog:
 
 dockers:
   - image_templates:
-      - scylladb/hydra-loaders:gemini-{{ .Tag }}
```

Comment:
Same question here:
Is it in sync with qa and sct ? @fruch

Reply:
Yes, @yaronkaikov already created dockerhub repository and added credentials in gemini for uploading to dockerhub